index.html

```diff
@@ -6,388 +6,492 @@
+              <time class="blog-card__date" datetime="2022-06-20">20 Jun 2022</time>
+              <h3 class="blog-card__card-title">Sonic live</h3>
+              <p class="blog-card__desc">
                 One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
                 nulla imperdiet...
               </p>
-              <a href="#">Read more</a>
+              <a class="blog-card__link button button--arrow-icon" href="#">Read more</a>
             </article>
           </li>
-          <li>
-            <article>
-              <span>NFT</span>
-              <img src="#" alt="Console" />
-              <p>Tony Nguyen</p>
-              <time datetime="2022-06-20">20 Jun 2022</time>
+          <li class="blog__list-item">
+            <article class="blog-card">
+              <span class="blog-card__label label">NFT</span>
+              <img class="blog-card__image" src="#" alt="Console" />
+              <p class="blog-card__article-writer">Tony Nguyen</p>
+              <time class="blog-card__date" datetime="2022-06-20">20 Jun 2022</time>
               <h3>Sonic live</h3>
-              <p>
+              <p class="blog-card__desc">
                 One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
                 nulla imperdiet...
               </p>
-              <a href="#">Read more</a>
+              <a class="blog-card__link button button--arrow-icon" href="#">Read more</a>
             </article>
           </li>
         </ul>
-        <a href="#">Read more</a>
+        <a class="blog__button button" href="#">Read more</a>
       </section>
-      <section>
-        <h2>About us</h2>
-        <p>
+      <section class="about-us">
+        <h2 class="about-us__title">About us</h2>
+        <p class="about-us__desc">
           Duis aute irure dolor in reprehenderit in voluptate velit esse cillum
           dolore eu fugiat nulla pariatur. Excepteur sint occae cat cupidatat
           non proident, sunt in culpa qui officia dese runt mollit anim id est
           laborum velit esse cillum dolore eu fugiat nulla pariatu epteur sint
           occaecat
         </p>
-        <ol>
-          <li>
-            <p>01</p>
-            <h3>High quality</h3>
-            <p>
+        <ol class="about-us__list">
+          <li class="about-us__list-item">
+            <p class="about-us__list-number">01</p>
+            <h3 class="about-us__list-title">High quality</h3>
+            <p class="about-us__list-desc">
               Lorem Ipsum is simply dummy text of the printing and Ipsum
               typesetting industry.
             </p>
           </li>
-          <li>
-            <p>02</p>
-            <h3>Big collection</h3>
-            <p>
+          <li class="about-us__list-item">
+            <p class="about-us__list-number">02</p>
+            <h3 class="about-us__list-title">Big collection</h3>
+            <p class="about-us__list-desc">
```

Comment:
It seems to me that such class naming is quite appropriate here, although it is recommended to avoid it. It looks logical and very understandable in the lists. So you're doing great!